migrations/app/ddl_migrations/ddl_functions/20250509183824_fn_get_origin_queue.up.sql

```diff
@@ -0,0 +1,394 @@
+            FROM mto_shipments ms
+            WHERE ms.move_id = moves.id
+        ) shipments ON TRUE
+        LEFT JOIN LATERAL (
+            SELECT MIN(ppm.expected_departure_date) AS earliest_expected_departure_date
+            FROM ppm_shipments ppm
+            JOIN mto_shipments ms ON ppm.shipment_id = ms.id
+            WHERE ms.move_id = moves.id
+        ) ppm_dates ON TRUE
+        LEFT JOIN LATERAL (
+            SELECT
+                json_agg(
+                    json_build_object(
+                        ''id'', si.id,
+                        ''status'', si.status,
+                        ''re_service'', json_build_object(
+                            ''code'', rs.code
+                        )
+                    )
+                )::JSONB AS mto_service_items
+            FROM mto_service_items si
+            JOIN re_services rs ON si.re_service_id = rs.id
+            WHERE si.mto_shipment_id IN (
+                SELECT ms.id FROM mto_shipments ms WHERE ms.move_id = moves.id
+            )
+        ) service_items ON TRUE
+        WHERE moves.show = TRUE ';
+
+    IF user_gbloc IS NOT NULL THEN
+        sql_query := sql_query || '
+        AND EXISTS (
+            SELECT 1 FROM mto_shipments ms
+            WHERE ms.move_id = moves.id
+            AND (
+                (ms.shipment_type != ''HHG_OUTOF_NTS'' AND move_to_gbloc.gbloc = $1)
+                OR (ms.shipment_type = ''HHG_OUTOF_NTS'' AND orders.gbloc = $1)
+            )
+        )';
+    END IF;
+
+    IF customer_name IS NOT NULL THEN
+        sql_query := sql_query || ' AND (
+            service_members.first_name || '' '' || service_members.last_name ILIKE ''%'' || $2 || ''%''
+            OR service_members.last_name || '' '' || service_members.first_name ILIKE ''%'' || $2 || ''%''
+            OR service_members.last_name || '', '' || service_members.first_name ILIKE ''%'' || $2 || ''%''
+        )';
+    END IF;
+
+    IF edipi IS NOT NULL THEN
+        sql_query := sql_query || ' AND service_members.edipi ILIKE ''%'' || $3 || ''%'' ';
+    END IF;
+
+    IF emplid IS NOT NULL THEN
+        sql_query := sql_query || ' AND service_members.emplid ILIKE ''%'' || $4 || ''%'' ';
+    END IF;
+
+    IF m_status IS NOT NULL THEN
+        sql_query := sql_query || ' AND moves.status = ANY($5) ';
+    END IF;
+
+    IF move_code IS NOT NULL THEN
+        sql_query := sql_query || ' AND moves.locator ILIKE ''%'' || $6 || ''%'' ';
+    END IF;
+
+    IF requested_move_date IS NOT NULL THEN
+        sql_query := sql_query || '
+        AND (
+            shipments.earliest_requested_pickup_date::DATE = $7::DATE
+            OR ppm_dates.earliest_expected_departure_date::DATE = $7::DATE
+            OR EXISTS (
+                SELECT 1 FROM mto_shipments ms2
+                WHERE ms2.move_id = id
+                AND ms2.shipment_type = ''HHG_OUTOF_NTS''
+                AND ms2.requested_delivery_date::DATE = $7::DATE
+            )
+        )';
```

Comment:
Did you mean `moves.id`?
```suggestion
            OR EXISTS (
                SELECT 1 FROM mto_shipments ms2
                WHERE ms2.move_id = moves.id
                AND ms2.shipment_type = ''HHG_OUTOF_NTS''
                AND ms2.requested_delivery_date::DATE = $7::DATE
            )
        )';
```

Reply:
Actual question, I didn't see anything breaking cause of this

Reply:
This should be `moves.id` - you're correct

Reply:
Fixed in [this commit](https://github.com/transcom/mymove/pull/15457/commits/ef6ba44930fa5f7ba8545abb0fbbdccd92d5a823)